block/block.go

```diff
@@ -193,7 +193,7 @@ func GenesisBlockInit() (*Block, error) {
 		},
 	}
 
-	rewardAddress, _ := ToScriptHash("NcX9BWx5uxsevCZ2MUEbBJGoYGSNCuJJpf")
+	rewardAddress, _ := ToScriptHash("NKNU6H1TEckb3yMaSpmyyAbShh7g7GL7xipJ")
```

Comment:
How do we get this special string?

Reply:
The programHash of "NcX9BWx5uxsevCZ2MUEbBJGoYGSNCuJJpf" is "0xb628ea5423e34312a0f0ed29ed0a746adae214ee". Legacy address from first balance-dev branch commit.
Since wallet prefix changed from 'N' to "NKN", "NKNU6H1TEckb3yMaSpmyyAbShh7g7GL7xipJ" is the new address with same programHash.

Reply:
I see, so it's the miner pool's address